### PR TITLE
Feature/card layout

### PR DESCRIPTION
**Pull request by YarynaPuhach**
https://app.clickup.com/30346475/inbox?tab=important

### CODE REVIEW

File: StylingMethods.jsx
Comment (TideTwo) [inline StylingMethods.jsx:1]: this file is no longer needed

File: src/Globals.d.ts
Comment (TideTwo) [inline src/Globals.d.ts:1]: this should not be deleted

File: src/typings.d.ts
Comment (TideTwo) [inline src/typings.d.ts:1]: not needed

File: tsconfig.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "extends": "@mate-academy/students-ts-config",
-  "include": ["src"],
+  "include": [
+    "src"
+  ],
   "compilerOptions": {
     "sourceMap": false,
-    "types": ["node", "cypress"],
-    "plugins": [{ "name": "typescript-plugin-css-modules" }]
```
Comment (TideTwo) [inline tsconfig.json]: dont delete this plugin

**Review by TideTwo — CHANGES_REQUESTED** [review]

File: src/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -3,7 +3,10 @@ import styles from './Layout.module.scss';
 import { ProductsProvider } from '../../context/ProductsContext';
 
 type Props = {};
-export default function Layout({}: Props) {
+export default function Layout({ }: Props) {
```
Comment (JustinNowoczek) [inline src/components/Layout/Layout.tsx:6]: 2 functions?

**Pull request merged**